Source/Fuse.Controls.CameraView/iOS/CameraViewImpl.mm

```diff
@@ -400,6 +401,116 @@ void setCameraFacing(CameraView* cameraView, int cameraFacing, void(^onResolve)(
 		});
 	}
 
+
+	void setCameraFocusPoint(CameraView* cameraView, 
+		double x, double y, int cameraWidth, int cameraHeight, 
+		int isFocusLocked,
+		void(^onResolve)(int), void(^onReject)(NSString*)) {
+
+		AVState* avState = cameraView->avState;
+		dispatch_async(avState->queue, ^{
+
+			AVCaptureDevice* currentDevice = avState->deviceInput.device;
+
+			if(!currentDevice)
+				return;
+			
+			if([currentDevice isFocusPointOfInterestSupported] 
+				&& [currentDevice isFocusModeSupported:AVCaptureFocusModeContinuousAutoFocus]
+				&& [currentDevice isExposureModeSupported:AVCaptureExposureModeContinuousAutoExposure]
+				)
+			{
+				NSError* error;
+
+				//check to unlock
+				if (isFocusLocked != 1 && [currentDevice lockForConfiguration:&error])
+				{
```

Comment:
The other code in this file seems to use Egyptian braces, please be consistent with that. This applies elsewhere in this patch as well.

Reply:
Yeah, was a little confused between languages, one was using egyptian, the other wasn't, I personally like egyptian, so am gonna use it all round...